planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -736,9 +736,11 @@ void AvoidanceModule::fillShiftLine(AvoidancePlanningData & data, DebugData & de
    * Check whether the ego should avoid the front object. When the ego follows reference path,
```

Comment:
Should this description be fixed as well?

Reply:
Yes, I should. I'll fix in this PR.

Reply:
I fixed in https://github.com/autowarefoundation/autoware.universe/pull/3033/commits/9b52f29a649d4bcc0441790990bbf10cd027c81d